lua/staline.lua

```diff
@@ -88,9 +92,28 @@ local lsp_client_name = function()
             clients[#clients + 1] = client.name
         end
     end
-    return t.lsp_client_symbol .. table.concat(clients, ', ')
+    clients_name = table.concat(clients, ', ')
+
+    -- NOTE: Only show XX characters if the "clients_name" is too long
+    if name_max_lenght <= 0 then
+        return the_symbol .. clients_name
+    else
+        local clients_lenght = string.len(clients_name)
+        local clients_truncateName = ""
```

Comment:
`clients_truncated_name` instead of `client_truncatedName`.
Also small typos in `clients_lenght` and `name_max_lenght`

Reply:
Changes applied 👍